pkg/config/config.go

```diff
@@ -59,6 +59,9 @@ const (
 	// environment variables for the kube API
 	kubeAPIBurst = "KUBE_API_BURST"
 	kubeAPIQPS   = "KUBE_API_QPS"
+
+	terminationLogPathDefault = "/dev/termination-log"
```

Comment:
Does that work and make sense = writing to the `/dev` directory related to devices ?

Reply:
this is the kubernetes recommended way and default termination log location. can be overriden via deployment

Reply:
Indeed, sry, was not aware of a default. I though one always must specify `terminationMessagePath`. :-)